docroot/sites/default/settings.php

```diff
@@ -791,3 +791,4 @@
 # }
 
 require DRUPAL_ROOT . "/../vendor/acquia/blt/settings/blt.settings.php";
+$settings['install_profile'] = 'config_installer';
```

Comment:
Are you sure you want to hard wire this here?

Reply:
dang you drush... ![shake_fist](https://media.giphy.com/media/R54jhpzpARmVy/giphy.gif)